lock/src/main/java/org/infinispan/lock/logging/Log.java

```diff
@@ -37,4 +37,7 @@
 
    @Message(value = "Invalid scope for tag <clustered-lock>. Expected CACHE_CONTAINER but was %s", id = 29007)
    ClusteredLockException invalidScope(String scope);
+
+   @Message(value = "Clustered locks work in cluster mode", id = 29008)
```

Comment:
"Cannot create clustered locks when clustering is not enabled" or something like that. This more like https://github.com/infinispan/infinispan/blob/master/server/core/src/main/java/org/infinispan/server/core/logging/Log.java#L101

Reply:
ok